server/graphql/index.ts

```diff
@@ -85,6 +85,27 @@ export const rootValue = {
           space
         };
       });
+    },
+    spaces: (parent, { first = 10, skip = 0, id, spaces = [] }) => {
+      let allSpaces = Object.entries(clone(registrySpaces)).map(
+        (space: [any, any]) => {
+          space[1].id = space[0];
+          space[1].private = space[1].private || false;
+          space[1].about = space[1].about || '';
+          space[1].members = space[1].members || [];
+          return space[1];
+        }
+      );
+
+      if (id) {
+        spaces.push(id);
```

Comment:
When we use "id" there should be only 1 result, but with this implementation it's not always the case.